### PR TITLE
[Metallb] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
Target: https://metallb.universe.tf 
@fedepaol dose this look good?

### CODE REVIEW

File: projects/metallb/project.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+homepage: "https://metallb.universe.tf"
+language: go
+primary_contact: "fpaoline@redhat.com"
+auto_ccs:
+  - rbryant@redhat.com
```
Comment (fedepaol) [inline projects/metallb/project.yaml:5]: Please replace russell with obraunsh@redhat.com, as he's more active on the project.
Thanks!

**Comment by jonathanmetzman** [comment]
Why is this project critical?

**Comment by 0x34d** [comment]
> Why is this project critical?

MetalLB is a known load balancer, and OSS-Fuzz will only fuzz its routing protocols.

**Comment by jonathanmetzman** [comment]
Who uses it?

**Comment by fedepaol** [comment]
> Who uses it?

Hi, MetalLB maintainer here.
I can link the 2022 CNCF annual review:
https://github.com/cncf/toc/blob/main/reviews/2022-metallb-annual.md#adoption

TL;DR: the major kubernetes vendors (including Google and Red Hat) are offering metallb as part of their on premise solution. MetalLB is also used a lot by hobbyists that are running k8s on their home clusters.

CC @aojea / @michaelasp to shortcut on the Google side :-)

**Review by jonathanmetzman — APPROVED** [review]
LGTM
